pkg/command/plugin.go

```diff
@@ -386,6 +413,40 @@ func displayPluginListOutputSplitViewContext(availablePlugins []discovery.Discov
 	}
 }
 
+func displayInstalledPluginList(installedPlugins []cli.PluginInfo, writer io.Writer) {
+	var outputData [][]string
+
+	outputWriter := component.NewOutputWriter(writer, outputFormat, "Name", "Description", "Target", "Version", "Status" /*, "Context"*/)
+
+	for index := range installedPlugins {
+		newRow := []string{
+			installedPlugins[index].Name,
+			installedPlugins[index].Description,
+			string(installedPlugins[index].Target),
+			installedPlugins[index].Version,
+			installedPlugins[index].Status,
+		}
+		// TODO(khouzam): Fix after pre-alpha
+		// if installedPlugins[index].Scope == common.PluginScopeContext {
+		// 	newRow = append(newRow, installedPlugins[index].ContextName)
+		// }
+		outputData = append(outputData, newRow)
+	}
+
+	addDataToOutputWriter := func(output component.OutputWriter, data [][]string) {
```

Comment:
can we make it a util function so that we can also use it in other use cases to add data to output writer?

Reply:
You are right, we use this piece of code 4 times.  I will extract it as a function. 👍

Reply:
Actually, it turns out we don't need this loop at all: we should be using `outputWriter.AddRow()` directly.
I have updated that code.

I haven't cleaned up the other code that uses that loop as it should be done is a dedicated PR.